brainstormforce/ultimate-addons-for-gutenberg
# New: Review schema block.

### Description

Added a new review schema block.

### Types of changes




New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop review schema block.
- Add schema related attribute and check it on the schema tool.

### Checklist:
